packages/web/src/utils/launchpad-get-status-text.ts

```diff
@@ -26,13 +27,16 @@ export const getStatusText = (
 
   const formatTime = (days: number, hours: number, minutes: number) => {
     if (days > 0) {
-      return `in ${days} days ${hours} hours`;
+      return i18n?.t("Launchpad:common.time.inDaysHours", { days, hours });
```

Comment:
In my opinion, it seems better to take over a function that formats i18n as an argument than to use an i18n instance.
It seems like a good idea to have utility functions behave as pure functions.

```javascript
export const getStatusText = (
  type: PROJECT_STATUS_TYPE,
  startTime: string | undefined,
  endTime: string | undefined,
  format: (
    key: string,
    options?: { [key in string]: string | number },
  ) => string,
) => {
    ...
}
```